CHANGELOG.asciidoc

```diff
@@ -25,6 +25,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 
 This release also includes changes from <<release-3-6-5, 3.6.6>> and <<release-3-5-8, 3.5.8>>.
 
+* Changed `substring()` to use `startIndex, endIndex` instead of `startIndex, length`."
```

Comment:
We don't need to put this since it's not released yet, just update the documentation for it.

Reply:
Removed line